ivy/functional/frontends/jax/lax/linalg.py

```diff
@@ -38,3 +38,34 @@ def symmetrize(x):
         x = symmetrize(x)
 
     return ivy.eigh(x, UPLO=UPLO)
+
+  @to_ivy_arrays_and_back
+    def all_gather(x, axis_name, *, axis_index_groups=None, axis=0, tiled=False): 
```

Comment:
All functions must to have a test. You can see plenty of examples under the folder `ivy_tests`. As you can see there, all our tests are organized by categories. For example, for the jax linalg functions, the tests must be in this file [ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py)

Reply:
I recommend you to read our [contribution guide](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis). You will find step-by-step instructions on how to contribute to the codebase. You can also find function and tests examples 🙂